pkg/util/common.go

```diff
@@ -1246,5 +1246,51 @@ func IsPodReady(clientset *kubernetes.Clientset, namespace string, labelSelector
 		}
 	}
 	return true, nil
+}
+
+// GetOperatorNamespace returns the namespace of the synopsys operator based on the labels
+func GetOperatorNamespace(clientset *kubernetes.Clientset) (string, error) {
+	// check if operator is already installed
+	rcs, rcErr := ListReplicationControllers(clientset, "", "app=synopsys-operator,component=operator")
+	deployments, dErr := ListDeployments(clientset, "", "app=synopsys-operator,component=operator")
+	if rcErr != nil || dErr != nil || (len(deployments.Items) == 0 && len(rcs.Items) == 0) {
+		return "", fmt.Errorf("synopsys operator namespace not found")
+	}
+	return deployments.Items[0].Namespace, nil
```

Comment:
Need to check which list to return right?  Either RC or Deployments, but currently you are only returning deployments.  Should check with list is not empty and return that one.

Reply:
nice catch. Fixed it